jdi-light/src/main/java/com/epam/jdi/light/elements/complex/dropdown/DropdownExpand.java

```diff
@@ -36,6 +36,7 @@ public UIElement expander() {
     public WebList list() {
         return linkedList(listLocator, "list").setUIElementName(INNER);
     }
+    public boolean autoClose = false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4921674)